e3sm_diags/plot/cartopy/polar_plot.py

```diff
@@ -1,278 +0,0 @@
-        ax.set_extent([-180, 180, 50, 90], crs=ccrs.PlateCarree())
-    elif pole == "S":
-        ax.set_extent([-180, 180, -55, -90], crs=ccrs.PlateCarree())
-
-    cmap = get_colormap(cmap, parameters)
-
-    theta = np.linspace(0, 2 * np.pi, 100)
-    center, radius = [0.5, 0.5], 0.5
-    verts = np.vstack([np.sin(theta), np.cos(theta)]).T
-    circle = mpath.Path(verts * radius + center)
-    ax.set_boundary(circle, transform=ax.transAxes)
-
-    p1 = ax.contourf(
-        lon,
-        lat,
-        var,
-        transform=ccrs.PlateCarree(),
-        norm=norm,
-        levels=levels,
-        cmap=cmap,
-        extend="both",
-    )
-    ax.set_aspect("auto")
-    ax.coastlines(lw=0.3)
-
-    # Plot titles
-    if title[0] is not None:
-        ax.set_title(title[0], loc="left", fontdict=plotSideTitle)
-    if title[1] is not None:
-        ax.set_title(title[1], fontdict=plotTitle)
-    if title[2] is not None:
-        ax.set_title(title[2], loc="right", fontdict=plotSideTitle)
-
-    # Color bar
-    cbax = fig.add_axes((panel[n][0] + 0.35, panel[n][1] + 0.0354, 0.0326, 0.1792))
-    cbar = fig.colorbar(p1, cax=cbax)
-    w, h = get_ax_size(fig, cbax)
-
-    if levels is None:
-        cbar.ax.tick_params(labelsize=9.0, length=0)
-
-    else:
-        maxval = np.amax(np.absolute(levels[1:-1]))
-        if maxval < 10.0:
-            fmt = "%5.2f"
-            pad = 25
-        elif maxval < 100.0:
-            fmt = "%5.1f"
-            pad = 25
-        else:
-            fmt = "%6.1f"
-            pad = 30
-        cbar.set_ticks(levels[1:-1])
-        labels = [fmt % level for level in levels[1:-1]]
-        cbar.ax.set_yticklabels(labels, ha="right")
-        cbar.ax.tick_params(labelsize=9.0, pad=pad, length=0)
-
-    # Min, Mean, Max
-    fig.text(
-        panel[n][0] + 0.35,
-        panel[n][1] + 0.225,
-        "Max\nMean\nMin",
-        ha="left",
-        fontdict=plotSideTitle,
-    )
-    fig.text(
-        panel[n][0] + 0.45,
-        panel[n][1] + 0.225,
-        "%.2f\n%.2f\n%.2f" % stats[0:3],
-        ha="right",
-        fontdict=plotSideTitle,
-    )
-
-    # RMSE, CORR
-    if len(stats) == 5:
-        fig.text(
-            panel[n][0] + 0.35,
-            panel[n][1] + 0.0,
-            "RMSE\nCORR",
-            ha="left",
-            fontdict=plotSideTitle,
-        )
-        fig.text(
-            panel[n][0] + 0.45,
-            panel[n][1] + 0.0,
-            "%.2f\n%.2f" % stats[3:5],
-            ha="right",
-            fontdict=plotSideTitle,
-        )
-
-
-def plot(reference, test, diff, metrics_dict, parameter):
-    # Create figure, projection
-    fig = plt.figure(figsize=parameter.figsize, dpi=parameter.dpi)
-
-    # Create projection
-    if parameter.var_region.find("N") != -1:
-        pole = "N"
-        proj = ccrs.NorthPolarStereo(central_longitude=0)
-    elif parameter.var_region.find("S") != -1:
-        pole = "S"
-        proj = ccrs.SouthPolarStereo(central_longitude=0)
```

Comment:
TODO: this doesn't have a correspondence yet! `e3sm_diags/plot/utils.py` is currently expecting only to do a `PlateCarree` projection.